extensions/wikidata/tests/data/manifest/wikidata-manifest-v1.0.json

```diff
@@ -0,0 +1,166 @@
+{
```

Comment:
As I mentioned by email, if we think users are going to have to deal with manifests, we may want to think about using YAML instead of JSON. In addition to be easier for humans to edit, it allows comments.